conference/DS/ics.yml

```diff
@@ -0,0 +1,16 @@
+- title: ICS
+  description: ACM International Conference on Supercomputing
+  sub: DS
+  rank: B
+  dblp: ics
+  confs:
+    - year: 2024
+      id: ics24
+      link: https://ics2024.github.io/
+      timeline:
```

Comment:
Please check the format, each subitem in `timeline` must contain a `deadline` property, standalone `abstract_deadline` is not allowed.